new_dawn_server/users/api/tests/test_users_api.py

```diff
@@ -34,6 +38,26 @@ def setUp(self):
             "school": "NYU",
             "smoke": "Socially",
         }
+        image1 = SimpleUploadedFile("media/images/test.png", b"file_content", content_type="image/png")
```

Comment:
File sent over from front-end wouldn't be a simple uploaded file instance. It should be an inmemoryuploadedfile.

Reply:
SimpleUploadFile inherits from inMemoryUploadedFile though and behaves like regular file that has name, content and content type. This can be useful when we have more files to test with.
https://docs.djangoproject.com/en/2.1/_modules/django/core/files/uploadedfile/
https://stackoverflow.com/questions/11170425/how-to-unit-test-file-upload-in-django